frontend/src/RecipeBase/RecipeBase.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import './RecipeBase.css';
+import {MainImage} from './MainImage/MainImage.js';
+import {SpecsBar} from '../SpecsBar/SpecsBar.js';
+import {TagsBar} from '../TagsBar/TagsBar.js';
+import {MainBody} from './MainBody/MainBody.js';
+import {IngredientList} from './IngredientList/IngredientList.js';
+import {Directions} from './Directions/Directions.js';
+import {CommentSection} from './CommentSection/CommentSection.js';
+
+class Base extends Component {
```

Comment:
May be best change Base to RecipeBase. As we all discussed, some of these components might be best baked back into the RecipeBase markup unless they're reusable pieces that will own state

Reply:
good call, change made